src/javascript-wrapper.js

```diff
@@ -44,7 +44,12 @@ var fRunUpdater = function(updater, updaterFormals, vni) {
              if (newStateLm === oldOutputStateLm ||
                  _.isUndefined(oldOutputStateLm) && ! _.isUndefined(newStateLm)) {
                  // updater did not modify output state lm - update it now
-                 vni.outputState({data: results, groupLm: groupLm, lm: createLm()});
+                 if (vni.nodeInstance.isTranslator && ! _.isUndefined(results.id)) {
+                     var id = 'urn:local:'+results.id+":(translator:"+vni.nodeInstance.componentName+")"; 
```

Comment:
One other thing: I think the URN can be like urn:local:myTranslatorNode:42 , i.e., going MSB to LSB from left to right, without the parens and without the word 'translator:'.